.gitignore

```diff
@@ -50,22 +50,23 @@ env
 venv*
 env_py3
 envpy3
-env36
 local_config.py
 /superset_config.py
 /superset_text.yml
 superset.egg-info/
 superset/bin/supersetc
 tmp
 rat-results.txt
-superset/app/
 
 # Node.js, webpack artifacts, storybook
 *.entry.js
 *.js.map
 node_modules
 npm-debug.log*
-superset/static/assets
+superset/static/assets/*
+!superset/static/assets/.gitkeep
+superset/static/uploads/*
+!superset/static/uploads/.gitkeep
```

Comment:
This checks both directories into git, thus being empty default directories. Any content will be ignored.